spec/features/state_file/complete_intake_spec.rb

```diff
@@ -290,6 +290,10 @@
       choose I18n.t("general.negative")
       click_on I18n.t("general.continue")
 
+      expect(page).to have_text I18n.t("state_file.questions.federal_extension_payments.edit.title")
```

Comment:
if you pull in the recent changes to complete spec you can change this to which just adds a retry if the complete intake spec gets snagged on it 
```suggestion
      page_change_check(I18n.t("state_file.questions.federal_extension_payments.edit.title"))
```